databricks/koalas/groupby.py

```diff
@@ -2316,6 +2314,31 @@ def value_counts(self, sort=None, ascending=None, dropna=True):
         )
         return _col(DataFrame(internal))
 
+    def unique(self):
+        """
+        Return unique values in group.
+
+        Uniques are returned in order of unknown. It does NOT sort.
+
+        See Also
+        --------
+        databricks.koalas.Series.unique
+        databricks.koalas.Index.unique
+
+        Examples
+        --------
+        >>> df = ks.DataFrame({'a': [1, 1, 1, 2, 2, 2, 3, 3, 3],
+        ...                    'b': [1, 2, 2, 2, 3, 3, 3, 4, 4]}, columns=['a', 'b'])
+
+        >>> df.groupby(['a'])['b'].unique().sort_index()  # doctest: +SKIP
+        a
+        1    [1, 2]
+        2    [2, 3]
+        3    [3, 4]
+        Name: b, dtype: object
+        """
+        return self._reduce_for_stat_function(F.collect_set, only_numeric=False)
```

Comment:
Oh haha I didn't know it can be simple like this.

Reply:
cool.

Reply:
I was surprised too while I was investigating. haha